sources/HUBViewControllerImplementation.m

```diff
@@ -1314,6 +1337,103 @@ - (void)removeComponentWrapperFromLookupTables:(nullable HUBComponentWrapper *)c
     }
 }
 
+- (void)scrollToRootComponentAtIndex:(NSUInteger)componentIndex
+                      scrollPosition:(UICollectionViewScrollPosition)scrollPosition
+                            animated:(BOOL)animated
+                          completion:(void (^)())completionHandler
+{
+    NSUInteger const componentCount = (NSUInteger)[self.collectionView numberOfItemsInSection:0];
+    if (componentIndex == NSNotFound || componentIndex >= componentCount) {
```

Comment:
Normally, we assert - since it's an incorrect use of the API. Failing silently has the downside that no-one knows that they did something wrong, until we get a bug report 😄